src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -126,6 +114,26 @@ declare const __LOADER__IS_LAZY__: any;
     _currentScriptTag.parentNode!.insertBefore(_newScriptTag, _currentScriptTag);
   }
 
+  // We want to ensure to only add default integrations if they haven't been added by the user.
+  function setupDefaultIntegrations(config: any, SDK: any) {
+    const integrations: {name: string}[] = config.integrations || [];
+    const integrationNames = integrations.map(integration => integration.name);
+
+    // Add necessary integrations based on config
+    if (config.tracesSampleRate && integrationNames.indexOf('BrowserTracing') === -1) {
```

Comment:
integrations can be an function 😢 - we'll have to account for that use case as well.

Reply:
hmmm, good point! I would tend to ignore this case for now (=do not register default integrations if this is a function). WDYT? I think everything else would be a bit tricky, probably...

Reply:
yup i think its fine to ignore! it is pretty advanced config, so we can add a note in the docs about it.

Reply:
👍 updated this, also updating the script in sentry-javascript PR and adding a test for this here: https://github.com/getsentry/sentry-javascript/pull/7790/commits/386cd2b97c660f6d149c462439826b9c150156b8